ivy/functional/frontends/torch/tensor.py

```diff
@@ -937,10 +937,10 @@ def fix(self):
     def fix_(self):
         self.ivy_array = self.fix().ivy_array
         return self
-        
```

Comment:
A little oversight regarding whitespace and indentation that previously caused a lint error.